Soccer Match.rtf

```diff
@@ -7,6 +7,11 @@
 
 \f0\fs24 \cf0 The app we\'92re building is soccer game. we need 2 teams of 11 players. they will play in a match. the match will keep the score. at the end we will display who wins.\
 \
+struct Team \{\
+var players: [Player(name: string, goals: Int, country: string, age: Int)]\
+let position: String\
```

Comment:
I’d change the nomenclature here from `position` to `leagueStanding` or something because we already have a `position` for thé `Player` object and the two can be confusing